ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -278,7 +278,8 @@ final class SandboxServer(
           packageStore,
           config.eventsPageSize,
           metrics,
-          lfValueTranslationCache
+          lfValueTranslationCache,
+          !config.implicitPartyAllocation
```

Comment:
Please name this parameter.

In fact, please name all the parameters. :upside_down_face:

Reply:
Done.